types/lodash/scripts/package.json

```diff
@@ -3,8 +3,7 @@
   "name": "lodash-scripts",
   "version": "0.0.1",
   "scripts": {
-    "generate": "ts-node generate-fp",
```

Comment:
seems like this should change to be `"modules": "ts-node generate-modules"`, right?

Reply:
That's a matter of design decision.

Which do you think is the best?

### Option1 (current):
```
  "scripts": {
    "generate": "ts-node generate-modules && ts-node generate-fp"
  },
```

### Option2:
```
  "scripts": {
    "modules": "ts-node generate-modules",
    "fp": "ts-node generate-fp"
  },
```

### Option 2'
```
  "scripts": {
    "all": "npm run modules && npm run fp",
    "modules": "ts-node generate-modules",
    "fp": "ts-node generate-fp"
  },
```

## Things to consider - from my understanding
* Authors have to run both `generate-modules.ts` and `generate-fp.ts` to generate both `<module-name>.d.ts` and `fp/*.d.ts`
* The instructions are https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bf8afd29658567c0c5cd447b3e508a9ec9dcaaf3/types/lodash/readme.md#before-creating-a-pr
  * This is one of the reasons why I chose option1 - the README already instructs authors to run `npm run generate` and I thought it should contain all necessary scripts.
  * But of course it's also OK to rewrite README to instruct like "Re-generate the files by opening a terminal in the scripts directory and running `npm run all` for example if we choose option 2' above.

Reply:
I think your choice is good -- it's backward compatible and a small-ish change.